docs/content/specs-defs/specs/bicep/interfaces.md

```diff
@@ -275,3 +275,30 @@ Which returns a JSON-formatted output like:
 This interface is a **SHOULD** instead of a **MUST** and therefore the AVM core team have not mandated a interface schema to use.
 
 {{% /notice %}}
+
+## Zone-redundant resources
+
```

Comment:
Can we please add a few introductory sentences and explain/link to the difference between zone-redundant and zonal?

Reply:
Extended 🚀

Reply:
Looks great. Just provided a suggestion if makes sense